tests/virt_autotest/virt_utils.pm

```diff
@@ -146,14 +146,16 @@ sub update_guest_configurations_with_daily_build {
 sub clean_up_red_disks {
     my $wait_script           = "120";
     my $get_disks_not_used    = "ls /dev/sd* | grep -v -e \"/dev/sd[a].*\" -e \"/dev/sd[b-z].*[[:digit:]].*\"";
-    my $disks_not_used        = script_output($get_disks_not_used, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my @disks_not_used        = script_output($get_disks_not_used, $wait_script, type_command => 1, proceed_on_failure => 1);
```

Comment:
I will try again with something else instead of formatting disks on my env using script_output. Doubt that split needs to be called here. Do you have any experience in using this API? I will also sift its usage through master repo.

Reply:
What I encountered is different. It seems that what script_output returns is not a quoted string. Or although it is quoted string, elements in it are sperated by new line. So bash command may not handle it as usual as space delimited arguments.

Reply:
Right, because it returns exactly what the bash command returns and it's multiple lines output:
![image](https://user-images.githubusercontent.com/28305643/43723887-09855df6-9999-11e8-9412-53eb2fd7d9c1.png)
And this is a string, so if you put it into list context, it will be converted to array, first element of which contains whole output.
The scenario I've described above is sporadic and not easy to reproduce, but just be aware of it and consider implementing some validation in perl after output was received.

Reply:
The better solution would be to convert this into a pure-bash snippet IMHO.